### ISSUE TITLE
Failed to install (due to sip)

### ISSUE THREAD

**Issue by smnthermes** [GDGVIT/pip-gui#31]
> ERROR: Could not find a version that satisfies the requirement sip==4.19.8 (from pipgui) (from versions: 5.0.0, 5.0.1, 5.1.0, 5.1.1, 5.1.2, 5.2.0, 5.3.0, 5.4.0)
> ERROR: No matching distribution found for sip==4.19.8 (from pipgui)

I have sip 4.19.21, which is newer than 4.19.8.

**Comment by Engensmax** [comment]
Same here. I have sip 6.0.0

**Comment by SaurusXI** [comment]
Yeah the project hasn't seen any improvements in a while. Looks like we can start with bumping dependencies.

Definition of done -
- [ ] Bump sip and other outdated dependencies
- [ ] Check for breakages
- [ ] Fix breakages if any

Should be a relatively easy fix, I'm leaving this open if anyone would like to contribute
